### PR TITLE
add meta data, schema and google analytic

### PR DESCRIPTION
**Pull request by hungpv-2151**

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  chainWebpack: (config) => {
+    config
+      .plugin('html')
+      .tap(args => {
+        args[0].meta = {
+          viewport: 'width=device-width, initial-scale=1.0',
+          'theme-color': '#02002E',
+          description: process.env.VUE_APP_WEB_DESCRIPTION,
```
Comment (kimnh-0823) [inline vue.config.js]: Add keywords

File: vue.config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          'og:url': process.env.BASE_URL,
+          'og:type': 'website',
+          'og:title': '',
```
Comment (kimnh-0823) [inline vue.config.js]: Missing title

File: vue.config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          'og:description': process.env.VUE_APP_WEB_DESCRIPTION,
+          'og:image': process.env.BASE_URL + 'favicon.ico',
+          'twitter:card': 'summary',
+          'twitter:description': process.env.VUE_APP_WEB_DESCRIPTION,
+          'twitter:title': '',
```
Comment (kimnh-0823) [inline vue.config.js]: add title

File: vue.config.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+          'twitter:image': process.env.BASE_URL + 'favicon.ico',
```
Comment (kimnh-0823) [inline vue.config.js]: `BASE_URL` should not include `/ ` at the end.

File: vue.config.js (hunk 1 above)
Comment (hungpv-2151) [reply vue.config.js]: Google does not use the keywords meta tag in web ranking
https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html

File: public/index.html [hunk 5]
```diff
@@ -1,56 +1,66 @@
...
 <html lang="en">
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="icon" href="<%= VUE_APP_WEB_BASE_URL %>favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (kimnh-0823) [inline public/index.html]: Add meta tag for keywords:
```html
    <meta name="keywords" content="<%= ...." />
```

File: .env.example [hunk 6]
```diff
@@ -1,2 +1,4 @@
 VUE_APP_GA_TRACKING_ID=
+VUE_APP_WEB_TITLE=
 VUE_APP_WEB_DESCRIPTION=
+VUE_APP_WEB_BASE_URL=
```
Comment (kimnh-0823) [inline .env.example]: Keep a blank line at the end of file

**Pull request merged**